.travis.yml

```diff
@@ -82,7 +82,7 @@ before_install:
         export PATH="$(brew --prefix ccache)/libexec:$PATH";
         BAR="pv -i 30";
         contrib/travis_fastfail.sh || exit 1;
-        brew tap staticfloat/julia;
+        brew tap staticfloat/julia > /dev/null;
```

Comment:
I don't think we should `> /dev/null` here; this means we won't catch errors like what brought us to this PR in the first place.  Travis auto-collapses the brew installation section anyway, since it's not in the main testing script section.

Reply:
So leave it?